oem/smc/drive.go

```diff
@@ -0,0 +1,62 @@
+package smc
+
+import (
+	"encoding/json"
+
+	"github.com/stmcginnis/gofish/redfish"
+)
+
+type Drive struct {
+	redfish.Drive
+	Oem DriveOem `json:"Oem"`
+}
+
+type DriveOem struct {
```

Comment:
Don't have these three be public structs. They can be used for unmarhsalling the `Drive` data, but they shouldn't be exposed beyond that.

Reply:
I have this staged and expose the attributes within via getters to keep them from being exposed.

```go
func (d Drive) Temperature() int {
        return d.smc.Oem.Supermicro.Temperature
}
```